### ISSUE TITLE
Investigate use of logicals in queries to `ala_occurrences()`

### ISSUE THREAD

**Issue by mjwestgate** [AtlasOfLivingAustralia/galah#25]
It is possible to break a query in `ala_occurrences()` by setting a large number of years, for example:
```
> ala_occurrences(taxa = select_taxa("galah"), filters = select_filters(year = c(1700:2020)))
Error in check_for_caching(taxa_query, filter_query, area_query, columns) : 
  Too many filters provided.
```
A more intuitive method would do something like this:
```
 ala_occurrences(taxa = select_taxa("galah"), filters = select_filters(year >= 1700))
```
But it's unclear to me whether it's possible to build code like that given the restrictions of the API.

**Comment by matildastevenson** [comment]
It is possible to do this with the API- the query would be something like `year:[1700 TO 2000]`. The challenge is in working out a sensible way for a user to provide this to `select_filters`

**Comment by matildastevenson** [comment]
Possibly a helper function similar to `exclude` would work. e.g. `select_filters(year = from_to(1700,2000))`.

**Comment by daxkellie** [comment]
Use of logicals in `select_filters` -- now `galah_filter` -- was implemented in galah 1.3.1
